compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -279,6 +279,13 @@ static std::optional<ArrayAttr> getPaddingConvSizes(
   return b.getI64ArrayAttr(paddingConvSizes);
 }
 
+static FailureOr<int64_t> getRank(Value in) {
```

Comment:
I think we should mark this with `[[maybe_unused]]` to prevent warnings about unused functions